src/components/Header/Header.jsx

```diff
@@ -1,6 +1,18 @@
-// import the React library here
-// import a css file containig article styles
+import React from 'react';
+import './Header.css';
+import '../Navigation/Navigation.css';
 
-// Create a Header function returning the HTML of header block
+const Header = () => (
+  <>
+    <header className="header">
+      <h1 className="header__title">Site Name</h1>
+      <nav className="navigation">
+        <a className="navigation__link" href="#about">About</a>
+        <a className="navigation__link" href="#services">Services</a>
+        <a className="navigation__link" href="#contact">Contact</a>
+      </nav>
```

Comment:
```suggestion
      <Navigation />
```
it should be separate component